src/bentoml/_internal/configuration/v1/__init__.py

```diff
@@ -64,7 +64,10 @@
 }
 _API_SERVER_CONFIG = {
     "workers": s.Or(s.And(int, ensure_larger_than_zero), None),
-    "timeout": s.And(int, ensure_larger_than_zero),
```

Comment:
This is a breaking change. We should include a migration or deprecation warning below.